DigitalLearningSolutions.Web/appsettings.Production.json

```diff
@@ -1,12 +1,12 @@
-{
-  "AllowedHosts": "*",
-  "ConnectionStrings": {
-    "DefaultConnection": ""
-  },
-  "CurrentSystemBaseUrl": "https://www.dls.nhs.uk",
-  "AppRootPath": "https://www.dls.nhs.uk/v2",
-  "LegacyLearningMenu": false,
-  "FeatureManagement": {
-    "Login": false
-  }
-}
+{

```

Comment:
Could we add the new property here with the empty string value, like we do for "DefaultConnection" (and for other appsettings), so it's easy to find what the new property name is from appsettings. Then, the devs only need to populate it from what they can get in Keeper.

Reply:
Just a minor thing - it looks like we've missed plain appsettings.json